mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -338,6 +349,24 @@ public List<Record> getParsedRecordsFromInitialRecords(List<InitialRecord> rawRe
       }).collect(Collectors.toList());
   }
 
+  private ParsedResult validateIf999ffFieldExistsOnInstanceCreateAction(JobExecution jobExecution, ParsedResult parsedResult) {
```

Comment:
@VRohach , I think it would be better to rename "validateIf999ffFieldExistsOnInstanceCreateAction" to "addErrorMessageWhen999ffFieldExists" because "validate" is better used for true/false results, and checking for CreateAction is not present in this method.

Reply:
@Aliaksandr-Fedasiuk  Good point! I will do it.

Reply:
@Aliaksandr-Fedasiuk  But there is a checking for CreateAction for Instance. 358 line: `&& createInstanceActionExists(jobExecution)`

Reply:
@Aliaksandr-Fedasiuk  I will rename on: "addErrorMessageWhen999ffFieldExistsOnInstanceCreateAction".